README.md

```diff
@@ -0,0 +1,32 @@
+# KCP API Catalog
+
+A library for kcp API catalog code.
+
+The API Catalog provides publicly available interfaces (APIs, code, CLIs) that allows users to curate a collection of available APIs (`APIExport`) in the KCP environment.
```

Comment:
You say "user" in a number of places in the README. Can you replace user with the personas that might perform the actions? It might be more than one sometimes, but clarifying the roles of the people who would typically do that thing helps others understand.